src/components/charts/other.js

```diff
@@ -0,0 +1,264 @@
+import Highcharts from 'highcharts'
+
+import Chart from './chart-abstract'
+
+import {
+  heatmap,
+  tilemap,
+  polar,
+  boxplot,
+  gauge,
+  pyramid,
+  wordcloud,
+  sankey
+} from '../../constants/other/default-options-other'
+
+import {
+  modes,
+  initialState,
+} from '../../constants/other/modes-options-other'
+
+
+export default class Other extends Component {
+  constructor(props) {
+    super(props);
+    this.state = initialState;
+
+    this.dropdownClickHandler = this.dropdownClickHandler.bind(this);
+  }
+
+  componentDidMount() {
+    this.initHeatmap();
+  }
+
+  initHeatmap() {
+    const options = heatmap;
+
+    this.setState({ options }, () => {
+      this.updateHeatmapConfiguration();
+    });
+  }
+
+  initTilemap() {
+    const options = tilemap;
+
+    this.setState({ options }, () => {
+      this.updateTilemapConfiguration();
+    });
+  }
+
+  initPolar() {
+    const options = polar;
+
+    this.setState({ options }, () => {
+      this.updatePolarConfiguration();
+    });
+  }
+
+  initBoxplot() {
+    const options = boxplot;
+
+    this.setState({ options }, () => {
+      this.updateBoxplotConfiguration();
+    });
+  }
+
+  initGauge() {
+    const options = gauge;
+
+    this.setState({ options }, () => {
+      this.updateGaugeConfiguration();
+    });
+  }
+
+  initPyramid() {
+    const options = pyramid;
+
+    this.setState({ options }, () => {
+      this.updatePyramidConfiguration();
+    });
+  }
+
+  initWordcloud() {
+    const options = wordcloud;
+
+    this.setState({ options }, () => {
+      this.updateWordcloudConfiguration();
+    });
+  }
+
+  initSankey() {
+    const options = sankey;
+
+    this.setState({ options }, () => {
+      this.updateSankeyConfiguration();
+    });
+  }
+
+  updateHeatmapConfiguration() {
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  updateTilemapConfiguration() {
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  updatePolarConfiguration() {
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  updateBoxplotConfiguration() {
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  updateGaugeConfiguration() {
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  updatePyramidConfiguration() {
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  updateWordcloudConfiguration() {
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  updateSankeyConfiguration() {
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  dropdownClickHandler(input) {
```

Comment:
If you want, you can change this method as I did for Line, Pie and Bar chart.
Instead of `input` param use `mode`.

Reply:
Yes, I will do that after all the modes will be implemented. I will create an Issue soon for applying your tooltip implementation for all the chart pages. At the moment this implementation is temporary...